test/shellcheck_failing_output.txt

```diff
@@ -0,0 +1,76 @@
+============================= test session starts ==============================
+platform linux2 -- Python 2.7.6, pytest-2.9.2, py-1.4.31, pluggy-0.3.1 -- /usr/bin/python
+cachedir: .cache
+rootdir: /home/a/opensource/pi-hole, inifile: 
+plugins: cov-2.3.0, bdd-2.17.0, xdist-1.14, testinfra-1.4.0
+collecting ... collected 7 items
+
+test/test_000_build_containers.py::test_build_pihole_image[test/debian.Dockerfile-pytest_pihole:debian] PASSED
+test/test_000_build_containers.py::test_build_pihole_image[test/centos.Dockerfile-pytest_pihole:centos] PASSED
+test/test_automated_install.py::test_setupVars_are_sourced_to_global_scope[debian] PASSED
+test/test_automated_install.py::test_setupVars_are_sourced_to_global_scope[centos] PASSED
+test/test_automated_install.py::test_setupVars_saved_to_file[debian] PASSED
+test/test_automated_install.py::test_setupVars_saved_to_file[centos] PASSED
+test/test_shellcheck.py::test_scripts_pass_shellcheck FAILED
+
+=================================== FAILURES ===================================
+_________________________ test_scripts_pass_shellcheck _________________________
+
+    def test_scripts_pass_shellcheck():
+        ''' Make sure shellcheck does not find anything wrong with our shell scripts '''
+        shellcheck = "find . -name 'update.sh' | while read file; do shellcheck \"$file\"; done;"
```

Comment:
This line controls what we shellcheck.  Eventually it should probably be *.sh but since #890 patched up update.sh I changed this from automated-install.sh to update.sh last night just go get it to be green.